dist/systemd/obsservicedispatch.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=OBS Service Dispatch
+After=network.target
+
+[Service]
+EnvironmentFile=/etc/sysconfig/obs-server
+Environment=logdir=/srv/obs/log
+Type=oneshot
+ExecStart=/bin/bash -c "/bin/echo -n 'Initializing obsservicedispatch'"
+ExecStartPre=/bin/mkdir -p $logdir
+ExecStart=bsuser=`/usr/bin/perl -I/usr/lib/obs/server -MBSConfig -e 'print ( $BSConfig::bsuser || "obsrun" )'`
+ExecStart=bsgroup=`/usr/bin/perl -I/usr/lib/obs/server -MBSConfig -e 'print ( $BSConfig::bsgroup || "obsrun" )'`
+ExecStart=chown $bsuser:$bsgroup "$logdir"
```

Comment:
IMO, this must be static configuration and shouldn't be done on each service start.